markdown/frameworks/src/javalin/JavalinHelloWorld.java

```diff
@@ -0,0 +1,21 @@
+package javalin;
+
+import io.javalin.Javalin;
+import io.javalin.http.Context;
+import io.javalin.http.Handler;
+import org.jetbrains.annotations.NotNull;
+
+public class JavalinHelloWorld {
+    public static void main(String[] args) {
+        Javalin app = Javalin.create().start(8080);
+
+        //localhost:8080
+        app.get("/", new Handler() {
+            @Override
+            public void handle(@NotNull Context ctx) throws Exception {
```

Comment:
Du sagst `Override`. Kommt die `NotNull` aus dem Framework, d.h. fordern die das oder hast Du das "freiwillig" eingebaut?

Reply:
Kommt aus dem Framework. 
Du kannst es aber auch weglassen. 
Wenn ich die Dokumentation richtig deute, ist das eh eine non-functional Annotation. 
Soll ich die rauswerfen? 

https://www.jetbrains.com/help/idea/nullable-and-notnull-annotations.html

Reply:
Wenn du das eh schon besprochen hast, werfe ich es raus.

Reply:
hmmm, lass drin jetzt.